src/Aspire.Hosting.RabbitMQ/README.md

```diff
@@ -0,0 +1,43 @@
+# Aspire Hosting library for RabbitMQ
+
+Provides extension methods and resources to add RabbitMQ to the Aspire AppHost.
+
+## Install the package
+
+In your AppHost project, install the `Aspire.Hosting.RabbitMQ` library with [NuGet](https://www.nuget.org):
+
+```dotnetcli
+dotnet add package Aspire.Hosting.RabbitMQ
+```
+
+## Adding RabbitMQ to the AppHost
+
+To add RabbitMQ to the AppHost use the `AddRabbitMQ` method. The first parameter is the Aspire resource name. The second optional parameter
+is the port that the RabbitMQ container will listen on. If the port is not provided the port will be randomly assigned.
```

Comment:
I'm not sure how much we need to describe about every parameter in the README.

Checkout the Qdrant README format:

https://github.com/dotnet/aspire/blob/c7073e6a0bca74cfd4316e06021d9c53f5577470/src/Aspire.Hosting.Qdrant/README.md?plain=1#L15-L25